course/src/main/res/values/strings.xml

```diff
@@ -47,4 +47,5 @@
     <string name="course_continue_with">Continue with:</string>
     <string name="course_continue">Continue</string>
     <string name="course_to_proceed">To proceed with \"%s\" press \"Next section\".</string>
-</resources>
\ No newline at end of file
+    <string name="course_navigation_dates" translatable="false">Dates</string>
```

Comment:
Why this is not translatable as well?

Reply:
replaced with `tools:ignore="MissingTranslation"`.